### ISSUE TITLE
Need a lot of features yet (especially who wants to use an external HD)

### ISSUE THREAD

**Issue by RafaelTR** [ventoy/Ventoy#221]
I don't know if it's the developer's intention to support external IDs, but it doesn't work.

External HD

Reading the files takes a long time because we have other aquivos. Partitioning a partition to the ISOs before or after the larger partition does not work. It could also have a way to install without formatting the device. I did this, but i still didn't have success with partitions.

Bootloader

My motherboard is old, it's EFI (it has no UEFI interface), I can only boot in UEFI mode with a forced bootloader. I could have an integration with Clover.

**Comment by RaXorX** [comment]
Yeah, having to format the device, and on top of that use an external device is quite a pain. Especially when you want to create an internal sysrescue drive for your system with a good grub bootloader and a nice looking menu.
